touchforms/backend/xformserver.py

```diff
@@ -223,8 +223,7 @@ def handle_request(content, server):
             return xformplayer.current_question(content['session-id'], override_state=override_state)
 
         elif action == xformplayer.Actions.HEARTBEAT:
-            return xformplayer.heartbeat(content['session-id'])
```

Comment:
@czue, before, the restarter.sh would hit this route and error because there was no session-id. now that we don't check for the session-id it will return a 200 status code. before we were checking for a 500 (or 400 i forget). definitely better to check for a 200 though so i'm all for this change